### ISSUE TITLE
selector-type-no-unknown allow for custom tags eg. angular material

### ISSUE THREAD

**Issue by rvarbanov** [stylelint/stylelint#1471]
I understand the usefulness of this rule and I would love to use it, but it would be very helpful if there is a way to add custom HTML tags that will be skipped by this rule 

> Which rule, if any, is this issue related to?

`selector-type-no-unknown`

> What CSS is needed to reproduce this issue?

```css
md-input-container {
  color: pink;
}
```

> Which version of stylelint are you using?

6.6.0

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

Yes, it's related to non-standard HTML.

**Comment by jeddy3** [comment]
Can you try to the [`ignoreTypes` secondary option](https://github.com/stylelint/stylelint/tree/master/src/rules/selector-type-no-unknown#optional-options) please, and let us know if that doesn't work for you?

**Comment by rvarbanov** [comment]
Thank you!

Do I feel stupid now... I even read about the options... My bad.
